charts/dapr/crds/configuration.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Seems like CRD shema has been changed from v1beta to v1.

```
Error: failed to install CRD crds/components.yaml: CustomResourceDefinition.apiextensions.k8s.io "components.dapr.io" is invalid: [spec.versions: Invalid value: []apiextensions.CustomResourceDefinitionVersion(nil): must have exactly one version marked as storage version, status.storedVersions: Invalid value: []string(nil): must have at least one stored version]
Makefile:198: recipe for target 'docker-deploy-k8s' failed
```

Reply:
Ah you fixed it 👍

Reply:
```
Error: failed to install CRD crds/components.yaml: CustomResourceDefinition.apiextensions.k8s.io "components.dapr.io" is invalid: spec.validation.openAPIV3Schema.type: Required value: must not be empty at the root
```

Generating CRD via controller-gen is much easier. See #2355

Reply:
Yeah using that.